.github/workflows/workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Status
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x]
```

Comment:
Node 14 is released, but hasn't entered LTS, I think we should test against that too.

Reply:
Node 14 LTS starts 2020-10-27 so maybe it's reasonable to already enable testing againts it although the documentation states that "We only support Long-Term Support versions of Node.".

Reply:
I feel like, while we only support LTS versions, if a version is Current and will become LTS, we should probably add it to tests so that we aren't surprised by something.